File: problems/i/isithalloween/isithalloween.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    month, day = input().split()
    day = int(day)

    if (month == "OCT" and day == 31) or (month == "DEC" and day == 25):
        print("yup")
    else:
        print("nope")


if __name__ == "__main__":
    main()

File: problems/i/isithalloween/isithalloween.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <iostream>
#include <string>

using namespace std;

int main() {
    int day;
    string month;

    cin >> month >> day;

    if ((month == "OCT" && day == 31) || (month == "DEC" && day == 25)) {
        cout << "yup" << endl;
    }
    else {
        cout << "nope" << endl;
    }
}